templates/navigation-bar-new.twig

```diff
@@ -39,8 +40,23 @@
 			</span>
 		</button>
 	</div>
+
 	{% include 'search-form.twig' %}
-	{% include 'language-switcher.twig' %}
+
+	{% if site_languages is not empty %}
+	{% set current_lang = site_languages|filter(i => i.active)|first %}
+		<button
+			class="nav-languages-toggle"
+			type="button"
+			aria-label="{{ __( 'Choose language', 'planet4-master-theme' ) }}"
+			aria-expanded="false"
```

Comment:
I don't think these `data-bs` attributes are needed anymore, they were used for the toggling action... And the `aria-expanded` attribute can probably also be removed!
Not sure about `aria-label` though, maybe it's still useful for accessibility? 🤔